src/makeCalculator.js

```diff
@@ -33,11 +33,56 @@
  *
  * calculator.result === -28
  *
- *
  * @return {object}
  */
 function makeCalculator() {
-  // write code here
+  let result = 0;
+
+  const add = num => {
+    result += num;
+
+    return makeCalculator();
+  };
+
+  const subtract = num => {
+    result -= num;
+
+    return makeCalculator();
+  };
+
+  const multiply = num => {
+    result *= num;
+
+    return makeCalculator();
+  };
+
+  const divide = num => {
+    result /= num;
```

Comment:
There is no check for division by zero. Add validation to prevent undefined behavior.